armada/handlers/armada.py

```diff
@@ -116,6 +113,18 @@ def sync(self):
             if chart.release_name is None:
                 continue
 
+            # retrieve appropriate timeout value if 'wait' is specified
```

Comment:
we should be able to break this down. a little. 
why are you converting self.timeout to int?
getattr(chart, "var", None)